NitroxClient/Debuggers/EntityDebugger.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections.Generic;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.DataStructures;
+using UnityEngine;
+
+namespace NitroxClient.Debuggers
+{
+    public class EntityDebugger : BaseDebugger
+    {
+        public EntityDebugger() : base(200, null, KeyCode.E, true, false, false, GUISkinCreationOptions.DERIVEDCOPY)
+        {
+            ActiveTab = AddTab("EntityDebugger", RenderEntityDebugger);
+        }
+
+        private void RenderEntityDebugger()
+        {
+        }
+
+        public override void OnGUIExtra()
+        {
+            foreach (KeyValuePair<NitroxId, GameObject> gameObjectPairs in NitroxEntity.GetGameObjects())
+            {
+                NitroxId id = gameObjectPairs.Key;
```

Comment:
Do you need to initialize a new variable for a one time use?

Reply:
I guess not? Just seemed more consistent to have both vars up front.

Reply:
Then I think it would be the best to use `=>`